UI/forms/OBSBasicSettings.ui

```diff
@@ -2910,7 +2964,7 @@
                               </sizepolicy>
                              </property>
                              <property name="currentIndex">
-                              <number>1</number>
+                              <number>0</number>
```

Comment:
Was this a Qt Designer change or something else?

Reply:
This was me trying to reset all stacked widgets to their default tab.

As far as I can tell that widget _should_ be on index 0 (The radio group) by default, so this is also a regression fix.